src/utils.js

```diff
@@ -196,7 +196,10 @@ const createLambda = async (lambda, config) => {
     Timeout: config.timeout,
     Environment: {
       Variables: config.env
-    }
+    },
+    ...(config.vpcConfig && {
+      VpcConfig: config.vpcConfig
+    })
   }
```

Comment:
if there is vpcConfig, it will add 'VpcConfig' to the object, otherwise, it wont. 
The aws-sdk will not config any vpc stuff if there is no VpcConfig as part of the params object